content/en/docs/reference/android.md

```diff
@@ -44,8 +44,12 @@ When an authenticator is not persistently linked, a QR code must be scanned on e
 
 - The [Device Public Key (DPK)](../terms#device-public-key-dpk) extension is supported in beta, but is currently gated behind a flag in Chrome. Developers can enable `chrome://flags/#enable-experimental-web-platform-features` to experiment with DPK on Android, or on desktop Chrome when using [Cross-Device Authentication](../terms#cross-device-authentication-cda) with an Android device.
 
-## Resources
+## Official Resources
 
 - [Credential Manager API](https://developer.android.com/training/sign-in/passkeys)
 - [Security of Passkeys in the Google Password Manager](https://security.googleblog.com/2022/10/SecurityofPasskeysintheGooglePasswordManager.html)
 - [Sample app](https://github.com/android/identity-samples/tree/main/Fido2)
+
+## Community Resources
```

Comment:
Take a look at what happens if you set this header to `###` instead of `##`. Ideally we see an indented link to it under Resources like "Cross-Device Authentication" currently does under "Overview"...

![Screenshot 2023-04-11 at 7 12 18 AM](https://user-images.githubusercontent.com/5166470/231190633-94c4fc4c-5b5b-4d71-9349-2fafbed78096.png)

...but if it's not shown either I'd suggest it's fine.